examples/realms/applications/main.tf

```diff
@@ -115,6 +115,18 @@ resource "tozny_realm_application" "jenkins_oidc_application" {
   oidc_root_url = "https://jenkins.acme.com"
 }
 
+# A resource for creating an application role
```

Comment:
this example needs to move into a separate folder and path, leaving the current application example as is

Reply:
Where do you want it?

Reply:
`examples/realms/applications/roles/main.tf`

Reply:
done